src/views/RuleEngine/components/FromSelect.vue

```diff
@@ -7,6 +7,7 @@
     @keyup.enter="handleTopicInput"
     @blur="handleTopicInput"
     @change="handleSelectedChanged"
+    :class="{ 'is-mini': mini }"
```

Comment:
No need to set the type mini?

Reply:
Too lazy to rewrite styles. . Probably large or small edit box  :p